api/jobs/updateGeoIp.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const job = require('../parts/jobs/job.js'),
+    log = require('../utils/log.js')('job:updateGeoIp'),
+    { exec } = require('child_process');
+
+/** Class for the job of updating geo-ip's **/
+class UpdateGeoIP extends job.Job {
+    /**
+     * Run the update geo-ip job
+     * @param {done} done callback
+     */
+    run(done) {
+        log("Updating geo-ip's..");
+        exec('countly update geoip', (err) => {
+            if (err) {
+                log("Error:" + err);
+                return;
```

Comment:
incase of error, done callback will never be called.
is it the required behaviour?
shouldn't it call the done callback?

Reply:
https://github.com/Countly/countly-server/blob/428535f64ace72224f3d5db3c4d178b4bae04c3f/api/parts/jobs/job.js#L804

Reply:
is there no need for "countly update devices"?